src/NextButton.tsx

```diff
@@ -21,6 +21,7 @@ export default function BackButtonWrapper({ to }: Props): JSX.Element {
   return (
     <Button
       className="next-button"
+      data-testid="next-button"
```

Comment:
must cleaner way for the test for the next-button to be picked up, great work 🙌✨